xcode/Subconscious/Shared/Components/AppTabView.swift

```diff
@@ -9,7 +9,7 @@ import SwiftUI
 import ObservableStore
 import Combine
 
-enum AppTab: String {
+enum AppTab: String, Hashable {
```

Comment:
Added Hashable conformance to several action types so that we could check them for equality in tests.